### PR TITLE
fish 3.2.0

### PR DESCRIPTION
**Pull request by lithammer**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by zanchey** [comment]
3.2.0 needs the codesigning patch removed.

File: Formula/fish.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Fish < Formula
   desc "User-friendly command-line shell for UNIX-like operating systems"
   homepage "https://fishshell.com"
-  url "https://github.com/fish-shell/fish-shell/releases/download/3.1.2/fish-3.1.2.tar.gz"
-  sha256 "d5b927203b5ca95da16f514969e2a91a537b2f75bec9b21a584c4cd1c7aa74ed"
+  url "https://github.com/fish-shell/fish-shell/releases/download/3.2.0/fish-3.2.0.tar.xz"
+  sha256 "4f0293ed9f6a6b77e47d41efabe62f3319e86efc8bf83cc58733044fbc6f9211"
   license "GPL-2.0"
```
Comment (carlocab) [inline Formula/fish.rb:6]:
```suggestion
  license "GPL-2.0-only"
```
Most of the source headers specify only GPLv2, and not GPLv2+. (e.g., see `src/fish.cpp`)

**Comment by lithammer** [comment]
Removed code signing patch and updated the license.

**Review by dawidd6 — DISMISSED** [review]

**Comment by lithammer** [comment]
🤔

**Comment by carlocab** [comment]
Yea, GitHub Actions isn't really working at the moment. I've reported it to GitHub, but it may take a bit for them to get back to me.

**Comment by domoritz** [comment]
looks like GitHub actions work again and tests passed 🎉.

**Review by SMillerDev — DISMISSED** [review]

**Comment by SMillerDev** [comment]
Looks like the API is still broken :disappointed:

**Review by carlocab — DISMISSED** [review]

**Review by jason-yeo-paidy — DISMISSED** [review]

**Review by SMillerDev — DISMISSED** [review]

**Pull request closed without merging**